### PR TITLE
Add missing proofs

### PR DESCRIPTION
**Pull request by nanexcool**
`root` and `wards` added (the easy ones)

### CODE REVIEW

File: src/specs.md [hunk 1]
```diff
@@ -13,6 +13,41 @@ GNU Affero General Public License for more details.
 You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+```act
+behaviour wards of MkrAuthority
+interface wards(address usr)
+
+for all
+
```
Comment (kmbarry1) [inline src/specs.md]: Let's be consistent with spacing...I prefer the more compact version, personally:
```
for all
    May : uint256
```

File: src/specs.md [hunk 2]
```diff
@@ -13,6 +13,41 @@ GNU Affero General Public License for more details.
...
+  VCallValue == 0
+
+returns May
+```
+
+```act
+behaviour root of MkrAuthority
+interface root()
+
+types
```
Comment (kmbarry1) [inline src/specs.md:36]: `for all` was used above, here it's `types`...let's be consistent within a given file/repo, at least. I don't have a strong preference, I just care about consistency.

File: src/specs.md [hunk 3]
```diff
@@ -13,6 +13,41 @@ GNU Affero General Public License for more details.
...
+  Root : address
+
+storage
+  root |-> Root
```
Comment (kmbarry1) [inline src/specs.md:40]: Since you added a storage alias for `root`, let's change the other proofs to also use it (right now they hardcode `0`).

**Review by kmbarry1 — APPROVED** [review]
LGTM, just some minor nits to fix.

File: src/specs.md (hunk 1 above)
Comment (nanexcool) [reply src/specs.md]: fixed!

File: src/specs.md (hunk 2 above)
Comment (nanexcool) [reply src/specs.md:36]: should be fixed

File: src/specs.md (hunk 3 above)
Comment (nanexcool) [reply src/specs.md:40]: yup, they're all `root` now!

**Comment by kmbarry1** [comment]
LGTM, I'm gonna go ahead and merge, then try to get the proofs passing w/optimization enabled.

**Pull request merged**